seedlink_plotter/seedlink_plotter.py

```diff
@@ -546,90 +645,123 @@ def _parse_time_with_suffix_to_minutes(timestring):
         seconds = _parse_time_with_suffix_to_seconds(timestring)
     return seconds / 60.0
 
+def get_station_name(trace):
```

Comment:
this is never used and should be removed. if you need something like this, please don't use the seedlink naming convention as key names but rather the dot-separated SEED convention (e.g. `GR.FUR..HHZ`)